manifests/p/pyfa/pyfa/2.53.1/pyfa.pyfa.installer.yaml

```diff
@@ -0,0 +1,19 @@
+# Created with WinGet Automation using Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: pyfa.pyfa
+PackageVersion: 2.53.1
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2023-09-10
+Installers:
+- Architecture: x86

```

Comment:
```suggestion
- Architecture: x64
```

Reply:
Commit comment is supposed to be "Change Architecture", but somehow selecting this choice did not make it appear in the input bar, and clicking again in the input bar caused the change to commit.